.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         julia-version:
           - "1.6"
+          - "1"
```

Comment:
Why this?  Isn't this running twice on Julia v1.6 now?

Reply:
I was just putting this in for now with the assumption that 1.7 will be out relatively soon, and once that happens we'll want to test on both LTS and v1.